sui/wormhole/sources/governance_message.move

```diff
@@ -182,6 +175,9 @@ module wormhole::governance_message {
     /// Aborts if the VAA is not governance (i.e. sent from the governance
     /// emitter on the governance chain)
     fun assert_governance_emitter(wormhole_state: &State, verified_vaa: &VAA) {
+        // This state capability ensures that the current build version is used.
+        state::assert_current(wormhole_state);
```

Comment:
why doesn't this function just take a `StateCap`? Currently, when I read `verify_vaa` it's not immediately obvious to me that it's version protected, since that check is performed in `assert_governance_emitter` internally.

Proposal: let's remove `state::assert_current` altogether, and just propagate `StateCap`s. The `StateCap` should in turn be created at the "top-level" (i.e. entry point). That way, it will be easier to see that functions like `verify_vaa` are version protected.